src/volatile_memory.rs

```diff
@@ -954,31 +954,49 @@ where
         &self.bitmap
     }
 
-    /// Does a volatile write of the value `v` to the address of this ref.
+    /// Converts this to a [`VolatileSlice`](struct.VolatileSlice.html) with the same size and
+    /// address.
+    pub fn to_slice(&self) -> VolatileSlice<'a, B> {
```

Comment:
All the functions that don't check the invariants from the `SAFETY` section need to be marked as unsafe when calling the said functions.

Reply:
This function is already present in main branch and is not marked `unsafe` and so I didn't touch it. Though I updated all the helpers I have added now.